src/Core/hcoio_read_std_mod.F90

```diff
@@ -194,6 +194,7 @@ SUBROUTINE HCOIO_Read( HcoState, Lct, RC )
     LOGICAL                       :: FOUND
     LOGICAL                       :: IsModelLevel
     LOGICAL                       :: DoReturn
+    LOGICAL                       :: IGNORE
```

Comment:
Not sure why this was added. It doesn't seem to be used.

Reply:
Good catch. I think I was going to add something to ignore certain warnings, specifically the onces about data being treated as unitless but units are present in the file, e.g. mol mol-1 for restart variables. But then I decided against it, at least in this update.

Reply:
Resolved by https://github.com/geoschem/HEMCO/pull/298/commits/428c58da2dfced0a48f564504477a0e14249a93b